pkg/config/config_windows.go

```diff
@@ -38,6 +45,17 @@ func osinit() {
 	} else {
 		winutil.LogEventViewer(ServiceName, 0x8000000F, defaultConfdPath)
 	}
+
+	// Process Agent
+	if pd, err := winutil.GetProgramDataDir(); err == nil {
+		DefaultProcessAgentLogFile = filepath.Join(pd, "logs", "process-agent.log")
+	}
```

Comment:
Ca't this be moved to the code block above?

Reply:
Oh yeah good point, I didn't notice they were checking for the same things.